core/tests/protractor_utils/admin.js

```diff
@@ -31,6 +31,7 @@ var editConfigProperty = function(
     propertyName, objectType, editingInstructions) {
   general.waitForSystem();
   browser.get(general.ADMIN_URL_SUFFIX);
+  element(by.id('config')).click();
```

Comment:
We try to avoid accessing elements by ids as these can be changeable. Instead add a class to this relevant element called something like `protractor-test-admin-config-tab` and then use by.css() in the protractor file to access it.